### DISCUSSION TITLE
Semantic MediaWiki

### DISCUSSION THREAD

**Discussion by edbueme** [zim-desktop-wiki/zim-desktop-wiki#2521] [Ideas]
Can Zim be used to maintain a collection of wiki pages and work with Semantic Media Wiki?
I would like to have a conceptual scheme about the wiki pages that are under construction like a tree, where you can navigate through it, and have an insight about the whole wiki pages, is there any plugin to it?

**Comment by introt** [comment]
No, Zim's not really built for that - you're likely best off using the actual Media Wiki plugin. Do note that Zim Desktop Wiki is not connected to neither Media Wiki or the Kiwix Zim file format.
